tests/test_cnn.py

```diff
@@ -566,6 +560,7 @@ def test_find_duplicates_verbose_false(capsys):
 
 def test_scores_saving(cnn):
```

Comment:
the fixture need not be passed to the test since a new cnn object is being initialized locally.